patches/minecraft/net/minecraft/world/raid/Raid.java.patch

```diff
@@ -0,0 +1,35 @@
+--- a/net/minecraft/world/raid/Raid.java
++++ b/net/minecraft/world/raid/Raid.java
+@@ -18,6 +18,7 @@
+ import net.minecraft.advancements.CriteriaTriggers;
+ import net.minecraft.block.Blocks;
+ import net.minecraft.entity.Entity;
++import net.minecraft.entity.EntityClassification;
```

Comment:
No imports in patches, and are you even using it?

Reply:
fixed